### PR TITLE
docs: update css comments info and examples

### PR DESCRIPTION
**Pull request by dierat**
### Summary:
Part of https://app.shortcut.com/czi-edu/story/211537/move-css-and-js-comments-next-to-what-they-re-referencing which updates the CSS comment patterns away from using numbers to having comments just above the code they're referencing.

### Test Plan:
- verified the files produced by plop commands have the updated css comments
- verified in Visual Studio Code preview that the documentation looks right

css documentation preview in visual studio code

### CODE REVIEW

**Review by schakraborty22 — APPROVED** [review]
Looks good to me ,Thanks!

File: docs/CODE_GUIDELINES.md [hunk 1]
```diff
@@ -198,16 +198,18 @@ For example:
...
  */
-.button--full-width {
+.button--full-width.button--full-width {
   background: red;
-  margin-right: auto; /* 2 */
-  width: 100%; /* 1 */
-  height: 37px; /* 3 */
+  /* Pushes the button away from the other content because it lives in a flexbox container */
+  margin-right: auto;
+  /* Button should take up the full width of the container it lives within. */
```
Comment (jinlee93) [inline docs/CODE_GUIDELINES.md:209]: Can we remove this comment since it's self explanatory even though it is the docs might be referred to

Comment (dierat) [reply docs/CODE_GUIDELINES.md]: Removed! 👍

**Review by jinlee93 — APPROVED** [review]
Thanks for the docs and plop changes

**Pull request merged**